bin/graftM

```diff
@@ -129,60 +132,59 @@ if __name__ == '__main__':
                                         formatter_class=CustomHelpFormatter,
                                         epilog='''            
  ##################################################################################################################################
-                                                       ~~ GraftM "graft" ~~
+                                                         ~ GraftM graft  ~ 
  ##################################################################################################################################
 
- Use GraftM "graft" to search, align and classify reads using the following pipeline:
-    
-    Detect reads with HMM/Diamond --> Align to HMM --> Place in pre-built phylogenetic tree using Maximum Likelihood
+    graft is the main analysis pipeline of GraftM. To run graft, a GraftM package (gpkg) is required, the key components of which
+    are a HMM and DIAMOND database for searching  (with hmmsearch or DIAMOND) and a phylogenetic tree within which identified 
+    sequences are placed. Input sequence data may be provided in fasta (.fa) or fastq format, either gzipped (.fq.gz) or 
+    uncompressed. Protein or nucleotide sequences may be provided as input, and multiple datasets may be used in one run.    
```

Comment:
'graft' is the main analysis pipeline of GraftM. To run graft, a
GraftM package (gpkg) is required, the key components of which
are a HMM and DIAMOND database for searching (with hmmsearch or
DIAMOND) and a phylogenetic tree. Input sequence data may be
provided in fasta (.fa) or fastq format, either gzipped (.fq.gz)
or uncompressed. Protein or nucleotide sequences may be provided
as input and multiple datasets may be used in one run.